docs/guides/typescript.md

```diff
@@ -443,15 +443,9 @@ const bearStore = createStore<BearState>()((set) => ({
 }))
 
 function useBearStore(): BearState
-function useBearStore<T>(
```

Comment:
In this file, I removed the `equalityFn` instead of using `useStoreWithEqualityFn`. What is preferred, @dai-shi?

Reply:
Yeah, I like it.